src/auth/authController.ts

```diff
@@ -40,6 +40,7 @@ export class AuthController {
         return;
       }
     }
+
```

Comment:
move rows[0] to repository return statement. So business logic does not know about structure of database lib response